### PR TITLE
PLF-3964: Update resource bundle from crowdin for EXOGTN-1220 and EXOGTN...

### PR DESCRIPTION
**Pull request by maydt**
...-1280

### CODE REVIEW

File: web/portal/src/main/webapp/WEB-INF/classes/locale/portal/webui_en.properties [hunk 1]
```diff
@@ -264,7 +264,7 @@ UIUserProfileInputSet.label.HomeInfo=Home Info
 UIUserProfileInputSet.label.BusinessInfo=Business Info
 UIGroupMembershipForm.msg.user-not-exist=User "{0}" does not exist.
 UIGroupMembershipForm.msg.user-not-empty=The username can not be empty.
-UIGroupMembershipForm.msg.Invalid-char=The field "{0}" must {1}.
+UIGroupMembershipForm.msg.Invalid-char=The field "{0}" can only contain alpha,digit,comma,dash characters. The first and last character must be letters.
```
Comment (trangvh) [inline web/portal/src/main/webapp/WEB-INF/classes/locale/portal/webui_en.properties]: There is a space after each commas (checked in Crowdin). Is this a script bug?

Comment (maydt) [inline web/portal/src/main/webapp/WEB-INF/classes/locale/portal/webui_en.properties]: Yes, you're right, I'll update this file.

File: web/portal/src/main/webapp/WEB-INF/classes/locale/portal/webui_fr.properties [hunk 2]
```diff
@@ -237,7 +237,7 @@ FirstAndSpecialCharacterNameValidator.msg=Le champ "{0}" doit commencer par un c
 
 LastCharacterUsernameValidator.msg=Le champ "{0}" doit se terminer par un chiffre ou un caract\u00e8re.
 ConsecutiveSymbolValidator.msg=Le champ "{0}" ne peut pas contenir cons\u00e9cutivement les symboles "{1}" et "{2}".
-UsernameValidator.msg.Invalid-char=Seuls les minuscules, les chiffres, le point et le trait de soulignement sont autoris\u00e9s pour le champ "{0}".
+UsernameValidator.msg.Invalid-char=Seuls les caract\u00e8res alphanum\u00e9riques minuscule,les chiffres,le point et le trait de soulignement sont autoris\u00e9es pour le champ "{0}".
```
Comment (trangvh) [inline web/portal/src/main/webapp/WEB-INF/classes/locale/portal/webui_fr.properties]: Lack space after commas.

File: web/portal/src/main/webapp/WEB-INF/classes/locale/portal/webui_fr.properties [hunk 3]
```diff
@@ -759,55 +759,55 @@ UIWizardPageSelectLayoutForm.label.UIPageTemplateOptions=null
...
-UIWizardPageSetInfo.label.i18nizedLabel=Nom
+UIWizardPageSetInfo.label.i18nizedLabel=Nom Affich\u00e9
 UIWizardPageSetInfo.label.languages=#{word.language}
 UIWizardPageSetInfo.label.switchmode=Libell\u00e9 long
 UIWizardPageSetInfo.action.Save=#{word.save}
 UIWizardPageSetInfo.msg.null=Page inexistante
-UIWizardPageSetInfo.msg.node.deleted=Erreur de modification simultan\u00e9e : le noeud que vous avez s\u00e9lectionn\u00e9 a \u00e9t\u00e9 supprim\u00e9.
-UIWizardPageSetInfo.msg.navigation.deleted=Erreur de modification simultan\u00e9e : La navigation a \u00e9t\u00e9 supprim\u00e9e, vous devez donc fermer l'assistant.
+UIWizardPageSetInfo.msg.node.deleted=Erreur lors de la modification : le noeud que vous avez s\u00e9lectionn\u00e9 a \u00e9t\u00e9 supprim\u00e9.
+UIWizardPageSetInfo.msg.navigation.deleted=Erreur lors de la modification : La navigation a \u00e9t\u00e9 supprim\u00e9e,vous devez donc fermer l'assistant.
```
Comment (trangvh) [inline web/portal/src/main/webapp/WEB-INF/classes/locale/portal/webui_fr.properties]: Lack a space after the commas ", vous devez".

File: web/portal/src/main/webapp/WEB-INF/classes/locale/portal/webui_fr.properties [hunk 4]
```diff
@@ -935,29 +935,29 @@ UILogged.label.Abort=#{word.abort}
...
-UIForgetPasswordWizard.info=Nous nous excusons des ennuis que vous rencontrez \u00e0 cause de l'inaccessibilit\u00e9 ce site t\u00e9laire.<br/>Pour vite rem\u00e9dier au probl\u00e8me, veuillez suivre les \u00e9tapes de d\u00e9pannage en bas.<br /><br />1. Retrouver votre mot de passe: entrer votre <strong>nom d'utilisateur</strong>, puis cliquer sur \u00abEnvoyer\u00bb.<br/>2. Retrouver votre nom d'utilisateur: entrer <strong>votre adresse courrielle</strong>, puis cliquer sur \u00abEnvoyer\u00bb.<br/>
-UIForgetPasswordWizard.label.forgotpassword=Forgot my password
-UIForgetPasswordWizard.label.forgotusername=Forgot my username
+UIForgetPasswordWizard.info=Veuillez nous excuser de tout probl\u00e8me rencontrez d\u00fb a l'inaccessbilit\u00e9 \u00e0 ce site Web. < br /> Pour r\u00e9soudre ce probl\u00e8me rapidement,veuillez suivre les \u00e9tapes de d\u00e9pannage ci-dessous. < br /> < br /> 1. R\u00e9cup\u00e9rer votre mot de passe : entrez <strong>votre nom d'utilisateur</strong>,puis cliquez sur "Envoyer". <br/>2. R\u00e9cup\u00e9rer votre nom d'utilisateur : entrez <strong>votre adresse e-mail</strong>,puis cliquez sur "Envoyer". <br/>
```
Comment (trangvh) [inline web/portal/src/main/webapp/WEB-INF/classes/locale/portal/webui_fr.properties]: Lack a space after the commas.

**Pull request closed without merging**